@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardStorageUsage.vue

```diff
@@ -50,12 +55,18 @@ const data = computed<{
       return
     }
 
-    result.push({
+    progressBarItems.push({
       id: uuid,
       label: name_label,
-      value: percent,
+      current: percent,
+      total: 100,
     })
   })
-  return { result, maxSize, usedSize }
+
+  return {
+    progressBarItems,
+    maxSize: formatSizeRaw(maxSize, 2),
+    usedSize: formatSizeRaw(usedSize, 2),
```

Comment:
In XO 6, we display the value without decimal:

```suggestion
    maxSize: formatSizeRaw(maxSize, 0),
    usedSize: formatSizeRaw(usedSize, 0),
```